pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -223,7 +223,6 @@ func (e *cloudWatchExecutor) newSession(pluginCtx backend.PluginContext, region
 	}
 
 	return e.sessions.GetSession(awsds.SessionConfig{
-		HTTPClient: dsInfo.HTTPClient,
```

Comment:
nit but can we comment this line and link to the issue (and comment) where we explain the issue?